tools/bigquery-zos-mainframe-connector/README.md

```diff
@@ -1,326 +1,172 @@
-  --require_partition_filter <value>
-                           When specified, this flag determines whether to require a partition filter for queries over the supplied table. This flag only applies to partitioned tables. The default value is true.
-  --schema <value>         The path to a local JSON schema file or a comma-separated list of column definitions in the form [FIELD]:[DATA_TYPE],[FIELD]:[DATA_TYPE]. The default value is ''.
-  -t, --table              When specified, create a table. The default value is false.
-  --time_partitioning_expiration <value>
-                           An integer that specifies (in seconds) when a time-based partition should be deleted. The expiration time evaluates to the partition's UTC date plus the integer value. A negative number indicates no expiration.
-  --time_partitioning_field <value>
-                           The field used to determine how to create a time-based partition. If time-based partitioning is enabled without this value, the table is partitioned based on the load time.
-  --time_partitioning_type <value>
-                           Enables time-based partitioning on a table and sets the partition type. Currently, the only possible value is DAY which generates one partition per day.
-  --use_legacy_sql         When set to false, uses a standard SQL query to create a view. The default value is false (uses Standard SQL).
-  --view                   When specified, creates a view. The default value is false.
-  --dataset_id <value>     The default dataset to use for requests. This flag is ignored when not applicable. You can set the value to [PROJECT_ID]:[DATASET] or [DATASET]. If [PROJECT_ID] is missing, the default project is used. You can override this setting by specifying the --project_id flag. The default value is ''.
-  --debug_mode             Set logging level to debug. The default value is false.
-  --job_id <value>         The unique job ID to use for the request. If not specified in a job creation request, a job ID is generated. This flag applies only to commands that create jobs: cp, extract, load, and query.
-  --location <value>       A string corresponding to your region or multi-region location.
-  --project_id <value>     The project ID to use for requests. The default value is ''.
-  --synchronous_mode <value>
-                           If set to true, wait for the command to complete before returning, and use the job completion status as the error code. If set to false, the job is created, and successful completion status is used for the error code. The default value is true.
-  --sync <value>           If set to true, wait for the command to complete before returning, and use the job completion status as the error code. If set to false, the job is created, and successful completion status is used for the error code. The default value is true.
-  --stats_table <value>    tablespec of table to insert stats
+Check the [Using with BigQuery](./USING_WITH_BQ) guide.
 
```

Comment:
the link doesn't seem to work when I view it on github

Reply:
it should be working now